### ISSUE TITLE
modeling_bart疑似bug

### ISSUE THREAD

**Issue by HUSTHY** [RUCAIBox/ELMER#3]
` if exit_layers is not None:
                raise ValueError("The exit layers should be provided!")
            else:
                # whether a word exit in the current layer, True: exit, False: not exit
                # if True, copy the current hidden state to the cache (used in later layers)
                exited_signal = torch.eq(exit_layers, idx).unsqueeze(-1)
                exit_hidden_states = torch.where(exited_signal, hidden_states, exit_hidden_states)

                # if all words have exited, then break
                has_all_exited = has_all_exited | exited_signal
                if torch.all(has_all_exited):
                    break

                # only copy hidden states to the layer higher than the exit layer
                copy_signal = torch.le(exit_layers, idx).unsqueeze(-1)
                hidden_states = torch.where(copy_signal, exit_hidden_states.detach(), hidden_states)`

if分支可能有bug  应该修改为
`if exit_layers is None:
                raise ValueError("The exit layers should be provided!")`
